lib/compiler/compiler.ts

```diff
@@ -16,9 +18,14 @@ export async function compileAndSave(config: Config) {
 }
 
 export async function compile(config: Config) {
+  // Added support for backwards compatibillity - renamable node_modules path
+  const resolver = ImportsFsEngine()
+    .addResolver(resolvers.BacktrackFsResolver(config.npmPath));
+  const cache = await gatherSourcesAndCanonizeImports(findInputs(config.sourcesPath), process.cwd(), resolver);
```

Comment:
This line is quite long, having each argument in its own line would definitely increase readability

Reply:
I would suggest renaming `cache` to `sources` to match the rest of the project